BT2H-4 Implement Temp Sensor and LCD screen

ammonson3 opened the pull request:
DESCRIPTION: added temp sensor and lcd screen to arduino code

TESTS: hardware tests

Commit:
BT2H-4 Implement Temp Sensor and LCD screen

DESCRIPTION: added temp sensor and lcd screen to arduino code

TESTS: hardware tests

ykalra commented on Hardware/Temp___Lcd.ino:
```diff
@@ -0,0 +1,55 @@
+// Pass oneWire reference to DallasTemperature library
+DallasTemperature sensors(&oneWire);
+/**********************************************************/
+char array1[]=" SunFounder               ";  //the string to print on the LCD
```
array1 and array 2 could be deleted as they are unused in the code

ykalra commented on Hardware/Temp___Lcd.ino:
```diff
@@ -0,0 +1,55 @@
+/**********************************************************/
+char array1[]=" SunFounder               ";  //the string to print on the LCD
+char array2[]="hello, world!             ";  //the string to print on the LCD
+int tim = 250;  //the value of delay time
```
tim could also be deleted, if unused

ykalra reviewed (changes requested):
Apart from these small things, the code looks great! Thank you!

Commit:
BT2H-4 Implement Temp Sensor and LCD Screen

DESCRIPTION: Removed unused variables

TESTS: hardware tests

ammonson3 committed:
Add files via upload

BT2H-4 Implement Temp Sensor and LCD Screen

DESCRIPTION: LCD and Temp sensor circuit diagram

TESTS: Hardware tests

riyabhat2002 reviewed (approved):
Everything looks good! Thanks

riyabhat2002 reviewed (changes requested):
I tried making this circuit and the temperature sensor works but it doesn't show anything on the LCD screen. Could you put up the circuit that you used initially in which the LCD screen showed the temperature?
